
### Copyright 2025 Advanced Micro Devices, Inc.  All rights reserved.
### Licensed under the Apache License, Version 2.0 (the "License");
### you may not use this file except in compliance with the License.
### You may obtain a copy of the License at
###      http://www.apache.org/licenses/LICENSE-2.0
### Unless required by applicable law or agreed to in writing, software
### distributed under the License is distributed on an "AS IS" BASIS,
### WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
### See the License for the specific language governing permissions and
### limitations under the License.
diff --git a/multiurl/base.py b/multiurl/base.py
index f73c4c5..ae6b577 100644
--- a/multiurl/base.py
+++ b/multiurl/base.py
@@ -67,7 +67,7 @@ class DownloaderBase:
         parts=None,
         observer=_ignore,
         statistics_gatherer=_ignore,
-        progress_bar=progress_bar,
+        progress_bar=NoBar,
         resume_transfers=False,
         override_target_file=True,
         download_file_extension=None,
